services/headless-lms/server/src/domain/csv_export.rs

```diff
@@ -251,6 +251,7 @@ impl Write for CSVExportAdapter {
     }
 }
 
+/// Takes UnboundedReceiverStream-stream with AuthorizedResponse and returns UnboundedReceiverStream-stream with only Bytes
```

Comment:
Without this one, actix cannot stream our authorized streams as responses

also maybe an example